print/resources/inkstitch.js

```diff
@@ -17,6 +17,48 @@ function ping() {
    .fail(function() { $('#errors').attr('class', 'show') });
 }
 
+//function to chunk opd view into pieces 
+  // source: https://stackoverflow.com/questions/3366529/wrap-every-3-divs-in-a-div
+$.fn.chunk = function(size) {
+  var arr = [];
+  for (var i = 0; i < this.length; i += size) {
+    arr.push(this.slice(i, i + size));
+  }
+  return this.pushStack(arr, "chunk", size);
+}
+
+// build operator detailed view (opd)
+function buildOpd(thumbnail_size = $('#operator-detailedview-thumbnail-size').val() ) {
```

Comment:
Wow, I didn't even know complex code was possible in a default value like that.  Neat!